src/Directory.Packages.props

```diff
@@ -13,16 +13,20 @@
     <PackageVersion Include="Humanizer" Version="2.14.1" />
     <PackageVersion Include="Humanizer.Core" Version="2.14.1" />
     <PackageVersion Include="DotNetEnv" Version="2.5.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights" Version="2.21.0" />
+    <PackageVersion Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.15.0" />
```

Comment:
2.15.0 is a deprecated version of `Microsoft.ApplicationInsights.AspNetCore` and must be updated before merging.